### ISSUE TITLE
Improve get.sh

### ISSUE THREAD

**Issue by foxcpp** [foxcpp/maddy#164]
See https://github.com/foxcpp/maddy/issues/163#issuecomment-552694008

**Comment by foxcpp** [comment]
@Hexawolf I would like some reasoning on why that set line is "pure evil". I think otherwise [[1](http://redsymbol.net/articles/unofficial-bash-strict-mode/)].

**Comment by foxcpp** [comment]
```diff
-        if [ $SYSGOMAJOR -ne $WANTEDGOMAJOR ]; then
+        if [ "$SYSGOMAJOR" -ne "$WANTEDGOMAJOR" ]; then
             downloadgo=1
         fi
-        if [ $SYSGOMINOR -lt $WANTEDGOMINOR ]; then
+        if [ "$SYSGOMINOR" -lt "$WANTEDGOMINOR" ]; then
             downloadgo=1
         fi
-        if [ $SYSGOPATCH -lt $WANTEDGOPATCH ]; then
+        if [ "$SYSGOPATCH" -lt "$WANTEDGOPATCH" ]; then
             downloadgo=1
         fi
```
That doesn't seem to be necessary, these variables can't contain whitespace.

**Comment by foxcpp** [comment]
```diff
-if [ "$GOVERSION" == "" ]; then
+if [ -z "$GOVERSION" ]; then
     GOVERSION=1.13.4
 fi
-if [ "$MADDYVERSION" == "" ]; then
+if [ -z "$MADDYVERSION" ]; then
     MADDYVERSION=master
 fi
-if [ "$PREFIX" == "" ]; then
+if [ -z "$PREFIX" ]; then
     PREFIX=/usr/local
 fi
-if [ "$SYSTEMDUNITS" == "" ]; then
+if [ -z "$SYSTEMDUNITS" ]; then
     SYSTEMDUNITS=$PREFIX/lib/systemd
 fi
-if [ "$CONFPATH" == "" ]; then
+if [ -z "$CONFPATH" ]; then
     CONFPATH=/etc/maddy/maddy.conf
 fi
```
Is there some practical difference between these?

**Comment by foxcpp** [comment]
Remaining issues are addressed in 2696df025db1dc1af4ebd24d82e2908a6d590640, 556e432d634800e2bc4d3a6364662b58642b5723.
